Sources/LicenseList/View+Extensions.swift

```diff
@@ -23,7 +23,8 @@ extension View {
             }
         }
     }
-    
+
+    /// Sets the style for license views.
```

Comment:
How about explicitly specifying the scope, like `pickerStyle(_:)`?
```suggestion
    /// Sets the style for license views within this view.
```